frontend/src/components/CourseDescriptionPanel/CourseAttributes/CourseAttributes.tsx

```diff
@@ -31,6 +40,18 @@ const CourseAttributes = ({ course, courseCapacity }: CourseAttributesProps) =>
 
   const { study_level: studyLevel, terms, campus, code, school, UOC } = course;
 
+  const currentTerm = getMostRecentPastTerm(CURR_YEAR).T;
+
+  const recentTermNumber = terms?.length ? parseInt(terms[0].slice(1), 10) : currentTerm;
+
+  const yearCourseAvaliable = recentTermNumber > currentTerm ? CURR_YEAR - 1 : CURR_YEAR;
+
+  const updatedTerm = recentTermNumber > currentTerm ? recentTermNumber : currentTerm;
+
+  const teachingPeriod = updatedTerm > 0 ? 'T' + updatedTerm : 'U0';
```

Comment:
```suggestion
  const teachingPeriod = updatedTerm > 0 ? `T${updatedTerm}` : 'U0';
```
Changing this to be a template string, as eslint complains otherwise.